guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -761,6 +761,18 @@ class SampleEventObserverThatModifiesInputs
 
 15.8. Frequently update the third-party libraries used in the project/component to eliminate known vulnerabilities.
 
+15.9. LFI protection.
+
+15.9.1. SHOULD NOT trust user-submitted requests containing path and file name.
+
+15.9.2. SHOULD sanitize user-submitted path and file values to remove dot-dot-slash from the request.
+
+15.10. RCE protection.
+
+15.10.1. SHOULD avoid usage of eval(), passthru(), system(), shell_exec()
+
+15.10.2. SHOULD NOT directly pass user-submitted values to include*(), require*(), create_function(), fopen(), preg_replace()
+
```

Comment:
I'd recommend adding: 

> 15.10.3. SHOULD NOT use user-submitted data for variable functions.

Maybe a better way to phrase this?  I'm referring to` $userParam($userData);`

Reply:
@szurek nice addition. Just added.